modular_robot/revolve2/modular_robot/body/_module.py

```diff
@@ -1,9 +1,16 @@
 from __future__ import annotations
 
+from abc import ABC, abstractmethod
+from typing import Any
+
+from revolve2.simulation.scene import Color as SimulationColor
```

Comment:
This not possible, as the `modular_robot` package cannot depend on the `simulation` package.